Userland/Services/FileSystemAccessServer/main.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2021, timmot <tiwwot@protonmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <FileSystemAccessServer/ClientConnection.h>
+#include <LibCore/EventLoop.h>
+#include <LibCore/LocalServer.h>
+#include <LibGUI/Application.h>
+#include <LibIPC/ClientConnection.h>
+
+int main(int, char**)
+{
+    if (pledge("stdio recvfd sendfd rpath cpath wpath unix thread", nullptr) < 0) {
```

Comment:
Just a nitpick, but I'm not sure this needs `recvfd`, as it only sends FDs.

Reply:
I'm not certain why but I've tested without recvfd and it fails to start.